docs/sphinx/about/whats-new.txt

```diff
@@ -96,6 +102,11 @@ Other general improvements include:
 * fixes for the detection of csv pattern blocks by `FilePatternBlock`
 * bioformats_package.jar now includes bio-formats-tools as a dependency so
   ImageConverter, ImageFaker and ImageInfo classes are included in the bundle
+* initial work on
+  `Reader discoverability <https://github.com/openmicroscopy/design/issues/42>`_ now allows
+  optional/external Readers to be annotated into the Readers class and
```

Comment:
Not really. :smiley: (I'd love to be able to say something more helpful here but I'm too baffled; I'm happy to be outvoted if other readers don't share my confusion!)